### ISSUE TITLE
ML

### ISSUE THREAD

**Issue by uttamjaiswal86** [uttamjaiswal86/poc#14]
[Lecture_9_Companion.pdf](https://github.com/user-attachments/files/25000968/Lecture_9_Companion.pdf)
[Lecture_15_16_Companion_Doc.pdf](https://github.com/user-attachments/files/25000967/Lecture_15_16_Companion_Doc.pdf)
[Lecture_14_Companion.pdf](https://github.com/user-attachments/files/25000969/Lecture_14_Companion.pdf)
[Lecture_12_13_Companion.pdf](https://github.com/user-attachments/files/25000972/Lecture_12_13_Companion.pdf)
[Lecture_11_Companion.pdf](https://github.com/user-attachments/files/25000970/Lecture_11_Companion.pdf)
[Lecture_10_Companion.pdf](https://github.com/user-attachments/files/25000971/Lecture_10_Companion.pdf)

[Hand Written Derivation of tau learning rate and Steepest Descent.pdf](https://github.com/user-attachments/files/25000993/Hand.Written.Derivation.of.tau.learning.rate.and.Steepest.Descent.pdf)
[Golden section ratio.pdf](https://github.com/user-attachments/files/25000994/Golden.section.ratio.pdf)
[Derivation of learning rate for Gradient Descent.pdf](https://github.com/user-attachments/files/25000995/Derivation.of.learning.rate.for.Gradient.Descent.pdf)

![Feb VRE 2026.pdf](https://github.com/user-attachments/files/25159933/Feb.VRE.2026.pdf)

![LTA_26.pdf](https://github.com/user-attachments/files/25159932/LTA_26.pdf)
![E-Ticket_NF2AJ82W26581753900.pdf](https://github.com/user-attachments/files/25159953/E-Ticket_NF2AJ82W26581753900.pdf)

![2248936875.png](https://github.com/user-attachments/assets/bf00a8a7-2aa2-49ba-9bf6-f696c5108fd9)
